python/tvm/relay/op/_transform.py

```diff
@@ -94,7 +94,7 @@ def compute_strided_set(attrs, inputs, output_type):
 _reg.register_injective_schedule("strided_set")
 
 # layout_transform
-_reg.register_injective_schedule("layout_transform")
+_reg.register_strategy("layout_transform", strategy.layout_transform_strategy)
```

Comment:
Does this mean that layout_transform for all platforms stop to be scheduled through injective schedule?

Reply:
Hey @elvin-n. The plan is to make it so all implementations will have an injective like schedule.

For non-cuda targets it will still use an injective schedule via the generic fallback strategy. 

For cuda it will use the new schedule in the PR after using the autoinline injective schedule (I still need to figure out this part for cuda).

Reply:
It should now autoline and then apply new layout transform schedule rule. Though would probably need some additional testing